test/test_backend/db/test_utils.py

```diff
@@ -18,6 +18,8 @@
     select_failed_fetches, sql_query
 
 
+# todo: add datetime to setUp and tearDown: It might be possible, despite failsafes being in place to prevent refreshes
```

Comment:
Minor `todo`: It's possible for test assertion to fail if live data was inserted during the course of tests running, but I think given the nature of this test, and due to only 1 refresh being able to run at a time, that's a very low probability.